pkg/gencmd/app.go

```diff
@@ -0,0 +1,10 @@
+package gencmd
+
+import "github.com/izumin5210/grapi/pkg/cli"
+
+type CreateAppFunc func(*Ctx, *Command) (*App, error)
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type CreateAppFunc should have comment or be unexported